index.html

```diff
@@ -181,15 +181,15 @@ <h2>LETS CONNECT</h2>
                 </div>
```

Comment:
Kindly use a hx tag for the title of this block as it is a headline , I mean line 176  ` <p>Languages</p>`

Reply:
Thank you for notifying the correction. the issue has been resolved